content/posts/fixing-an-h264-encoding-bug.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: 'Fixing an H.264 encoding bug in the KittyCAD API'
+excerpt: |
+    Let's go through a bug deep within our WebRTC stream we solved this week!
+coverImage: 'meta-images/paultag-post-social.jpg'
+date: '2023-11-03T20:00:00Z'
+author:
+    name: Paul Tagliamonte
+    picture: '/paultag.jpg'
+ogImage:
+    url: 'meta-images/paultag-post-social.jpg'
```

Comment:
this doesn't exist yet, I didn't know how these were being generated

Reply:
I've had to make these manually for now (our auto-generated meta-images are still using out-of-date branding after the site flip), so I'll make this and add it to your website PR.